### ISSUE TITLE
Release to match Juno's Breakpoint Release

### ISSUE THREAD

**Issue by ChrisRackauckas** [Keno/Gallium.jl#187]
Juno is starting an ecosystem release which will enable the breakpoint GUI today. However, the ability to remove breakpoints requires https://github.com/Keno/Gallium.jl/pull/168 . I was wondering if a Gallium.jl release could follow as well since that would help the user experience. 

However, I do see that Gallium is not building on master. Is that due to some deps needing tags as well, or a more fundamental problem?

**Comment by josefsachsconning** [comment]
As far as I know, Gallium has not worked with the master branch of Julia since #153 on September 14.

**Comment by ChrisRackauckas** [comment]
Not talking about master branch of Julia. Julia v0.5 is fine.

**Comment by timholy** [comment]
Julia 0.6 has changed some of the fundamental types used for code introspection, so Gallium+dependents will need a **big** overhaul to support 0.6. I can imagine @Keno wants to wait until 0.6 is settling down (it's still not in feature freeze, seemingly).

**Comment by ChrisRackauckas** [comment]
>Julia 0.6 has changed some of the fundamental types used for code introspection, so Gallium+dependents will need a big overhaul to support 0.6. I can imagine @Keno wants to wait until 0.6 is settling down (it's still not in feature freeze, seemingly).

But do they not currently support v0.5? v0.6 is orthogonal to this issue.

**Comment by timholy** [comment]
Gallium is working for me on 0.5, though I have master checked out on many of the dependents.

**Comment by ChrisRackauckas** [comment]
>Gallium is working for me on 0.5, though I have master checked out on many of the dependents.

Okay, that's good to hear. So it's just a matter of tagging a bunch of things? For reference, this is what merged today:

https://github.com/JunoLab/atom-julia-client/pull/275#issuecomment-279498267

which will enable the breakpoint API, and these tags:

https://github.com/JuliaLang/METADATA.jl/pull/7945
https://github.com/JuliaLang/METADATA.jl/pull/7946

(along with release stuff for the Atom package manager) will enable it. Gallium.jl's master is currently required for the breakpoints to be unset because of #168, so although this will release, it won't have a great user experience yet. Since Gallium master is currently working (with a few tags on dependencies), what I am proposing is at least one last "pre-0.6" release so that way this is all complete.

Sorry if I wasn't explaining myself well earlier.

**Comment by timholy** [comment]
That's exciting news! Would indeed be great to get a new set of tags. @Keno?

**Comment by Keno** [comment]
Yes, I'm fine with a new set of tags, but as you point out 0.6 will require a massive refactor, so I'm not planning to update Gallium for 0.5 going forward.

**Comment by timholy** [comment]
Keno, if you've not used it before, `PkgDev.freeable()` can use used to print out a list of the gap (in number of commits) between the current master and the last tagged version. It's handy for quickly coming up with the list of packages that need tagging. For example:
```julia
julia> f = PkgDev.freeable()
Packages with a gap between HEAD and the most recent tag:
ASTInterpreter: 6
AbstractTrees: 13
AxisArrays: 3
COFF: 2
DWARF: 3
ELF: 2
FixedPointNumbers: 2
GLFW: 2
Gallium: 21
Gtk: 36
ImageAxes: 1
ImageFiltering: 9
ImageMagick: 1
ImageQuilting: 11
Immerse: 2
JuliaParser: 15
MachO: 3
ObjFileBase: 1
OffsetArrays: 1
Optim: -159
PkgDev: -13
Plots: 39
StructIO: 4
TerminalUI: 2
Unitful: 57
VT100: 10
VideoIO: 5
```
(The ones with a - mean I've got a branch checked out that is lacking a lot of stuff that's in current releases.)
